### DISCUSSION TITLE
Boot-SSD dead: Will MergerFS-pool be "recognized"?

### DISCUSSION THREAD

**Discussion by queen4me** [trapexit/mergerfs#1362] [General]
I'm using the mergerFS-Plugin for Openmediavault but my System-SSD died yesterday.
Is there any chance that a new installed OMV with mergerFS-Plugin will recognize the mergerFS-pool?
I think it won't but am I right that the mergerFS-pool consists of normal EXT4-formatted hdds so I don't have to fear that any data will be lost? Or is there anything mergerFS-related written to the hdds building the mergerFS-pool?

Thanks a lot for helping.

**Comment by trapexit** [comment]
This is really a OMV question and I don't work on or maintain the OMV plugin.

https://github.com/trapexit/mergerfs?tab=readme-ov-file#can-mergerfs-be-used-with-filesystems-which-already-have-data--are-in-use

No, mergerfs doesn't write config to the branches. It has no need to. There is no state to hold that is relevant. You just recreate the pool with the branches you want. That's kinda the whole point of the technology.

**Reply by queen4me** [reply]
Perfect and thanks a lot for your lightning fast answer on a saturday, trapexit.
I'm very happy that I made the decision to use mergerFS instead of RAID-5

**Reply by queen4me** [reply]
I did a new install on a new SSD, then I connected the former hdds of the mergerfs-pool, mounted them and created a new pool which is Shared an SMB.
I could access the shared mergerfs-pool on my windows client but I cant see any directories or files.
Are there some steps still missing?
I'm a bit nervous right now.

**Reply by trapexit** [reply]
As I said I'm not the maintainer or user of OMV. I can't help with setting up OMV. I can only speak to mergerfs and mergerfs doesn't mess with the data as I pointed out. Just mount the filesystem and look.
